shellphuzz

```diff
@@ -26,8 +26,13 @@ if __name__ == "__main__":
     parser.add_argument('-i', '--ipython', help="Drop into ipython after starting the fuzzer.", action='store_true')
     parser.add_argument('-T', '--tarball', help="Tarball the resulting AFL workdir for further analysis to this file -- '{}' is replaced with the hostname.")
     parser.add_argument('-m', '--helper-module', help="A module that includes some helper scripts for seed selection and such.")
+    parser.add_argument('--memory', help="Memory limit (in MB) to pass to AFL", default="8G")
```

Comment:
Can this be elaborated on? It says "in MB" and then gives a string with a "G" suffix. If you can provide suffixes, say that I guess?